docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -171,6 +171,24 @@ The minimal age of a query in the history before it is expired. An expired
 query is removed from the query history buffer and no longer available in
 the :doc:`/admin/web-interface`.
 
+``query.remote-task.enable-adaptive-request-size``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:** ``true``
+
+Enables dynamic split of ``taskUpdateRequest`` for schemas featuring large
+strings in table properties.
+
+``query.remote-task.guaranteed-splits-per-task``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Default value:** ``3``
+
+The number of guaranteed splits that will be sent in ``taskUpdateRequest`` for
```

Comment:
```suggestion
The number of guaranteed splits that is sent in ``taskUpdateRequest`` for
```

Reply:
are...

Reply:
Hahah ... one number is.. many splits are .. ;-)